src/src/tx_main.cpp

```diff
@@ -1510,6 +1512,9 @@ void loop()
         {
           // raw mavlink data - forward to USB rather than handset
           uint8_t count = CRSFinBuffer[1];
+          // Convert to CRSF telemetry where we can
+          convert_mavlink_to_crsf_telem(CRSFinBuffer, count, handset);
+          // If we have a backpack 
```

Comment:
I think this comment belongs one line down. The line directly below this is the write to the USB UART, not the backpack.